templates/skeleton/app/components/Aside.tsx

```diff
@@ -29,6 +35,23 @@ export function Aside({
   const {type: activeType, close} = useAside();
   const expanded = type === activeType;
 
+  useEffect(() => {
+    const abortController = new AbortController();
+
+    if (expanded) {
+      document.addEventListener(
+        'keypress',
```

Comment:
I couldn't get this to fire when pressing `Escape` in Chrome.. it turns out there are browser inconsistencies and [keypress is deprecated](https://developer.mozilla.org/en-US/docs/Web/API/Element/keypress_event) - `keydown` is MDN's recommended approach and got it working for me!